README.md

```diff
@@ -1,2 +1,7 @@
-# Anthos Bare Metal Edge Use Cases
-Edge computing is a distributed computing paradigm that brings computation and data storage closer to the location where the action is, to improve response times and save bandwidth. Multiple industries (e.g. Retail, Manufacturing and Energy) are facing a growing demand to provide localized, consistent, low latency services that expose compute services. Workloads range from order management, ML for images/traffic analysis/safety, device management like refrigeration, fryers, etc. [Anthos Bare Metal](https://cloud.google.com/anthos/clusters/docs/bare-metal/1.8) is a unique solution for deploying and managing workloads on clusters distributed across customer data centers and device fleets. 
+# Anthos Edge Use Cases
+
+The **Anthos Edge Use Cases** repository hosts a collection of samples/walkthroughs that enables learning and experimenting with [Anthos](https://cloud.google.com/anthos) for edge use cases. The core of repository is the [**Point of Sale**](point-of-sale-app) sample application around which the use cases are built.
```

Comment:
Thought: I was going to suggest linking to a Google Cloud documentation page that contains an overview specifically about Anthos Edge, but I realized we have no such page yet. 😳